tools/devops/automation/scripts/GitHub.psm1

```diff
@@ -264,7 +264,7 @@ class GitHubComments {
             }
         }
 
-        return $false
+        return $Env:IS_PR -eq "True"
```

Comment:
Note that I changed this function to check the branch name, which may be enough to properly detect if we're in a PR: https://github.com/xamarin/xamarin-macios/commit/f0d43752957c9c9b91e2b54e02b0cc74c2c1d165#diff-289ba09865f46d8622d3bf5deb2d0e4e86112883230745baf7fc2018f6b7f905R260-R265